prereise/gather/winddata/hrrr/grib.py

```diff
@@ -0,0 +1,111 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class GribRecordInfo:
+    """Class to hold metadata on a grib record. grib (General Regularly-distributed
```

Comment:
I'm not super familiar with GRIB, but following your link, it looks like it should be in all caps.